Casks/c/crystalviewer.rb

```diff
@@ -0,0 +1,26 @@
+cask "crystalviewer" do
+  version "11.1.0"
+  sha256 :no_check
+
+  url "https://crystalmaker.com/downloads/crystalviewer#{version.major}_mac.zip"
+  name "CrystalViewer"
+  desc "Interactive galleries of 3D crystal & molecular structures"
+  homepage "https://crystalmaker.com/crystalviewer/index.html"
+
+  livecheck do
+    url "https://crystalmaker.com/support/updates/VersionData-CVM.xml"
+    regex(/\b(\d+\.\d+\.\d+)\b/)
+    strategy :xml do |xml, regex|
+      xml.get_elements("versiondata/versionlist/version").map { |item| item.attributes["number"][regex, 1] }
```

Comment:
```suggestion
    strategy :xml do |xml|
      xml.get_elements("//version").map { |item| item.attributes["number"] }
```

Since the `number` attribute is the same value/format as the cask `version` (11.1.0), we can omit the regex for now. If the value changes in the future to include more than just the version string, we would need to use a regex then.

The XML only provides version information for the Mac version of CrystalViewer, so we can simply use `//version` to simply match `version` elements in the XML without being strict about the `versiondata` > `versionlist` > `version` hierarchy. The looseness shouldn't cause issues in this context but it may help the check to continue working if upstream makes negligible changes to the structure.

Reply:
Thank you for the instructions!